release-notes/README.md

```diff
@@ -4,6 +4,12 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 # Release notes
 
+## September 1, 2022
+
+### iOS Campaign Standard 3.0.0
+
+* Initial release to support [Adobe Campaign Classic workflows](https://github.com/Adobe-Marketing-Cloud/aep-sdks-documentation/tree/master/using-mobile-extensions/adobe-campaignclassic) for Adobe Experience Platform Mobile SDKs for iOS in Swift. This extension library is [available as an open source project on Github](https://github.com/adobe/aepsdk-campaignclassic-ios/).
+
```

Comment:
This release date is tentative. Will update the PR with correct dates once the release is done

Reply:
You need to update the release notes here - https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-campaignclassic/adobe-campaign-classic-release-notes as well.